Assets/Scripts/Models/Inventory.cs

```diff
@@ -191,4 +159,13 @@ private void ImportPrototypeSettings(int defaulMaxStackSize, float defaultBasePr
             Category = defaultCategory;
         }
     }
+
+    private void OnStackSizeChanged(Inventory inventory)
```

Comment:
I usually go for the Trigger prefix when sending an event, and On when receiving it. I think it makes it more clear.